FWCore/Utilities/interface/Exception.h

```diff
@@ -226,41 +208,22 @@ namespace cms {
 
   // template <typename E>
   // inline
-  // typename detail::Desired<E, detail::is_derived_or_same<Exception,E>::value>::type &
+  // typename detail::exception_type<E> &
```

Comment:
Unrelated to this PR, but do we still need this commented out code? The comment above refers to problems with GCC 3.2.3, but we are well beyond that by now.

Reply:
If we haven't needed it for this long then it is hard to justify that we'd need it in the future.

Reply:
Could you remove them then?